examples/tasks.go

```diff
@@ -43,4 +43,43 @@ func main() {
 	}
 	fmt.Println(task)
 
+	fmt.Println("Get a tree of tasks")
+	task_list_tree, err := ovc.GetTasks("taskState='Completed'", "", "10", "tree", "", "")
+	if err != nil {
+		fmt.Println("Error getting the task details with tree view ", err)
+	}
+
+	fmt.Println(task_list_tree.Members)
+
+	fmt.Println("Get a aggregate tree")
+	task_list_atree, err := ovc.GetTasks("taskState='Completed'", "", "", "aggregatedTree", "2", "2")
+	if err != nil {
+		fmt.Println("Error getting the task with status Completed and aggregatedTree", err)
+	}
+
+	fmt.Println(task_list_atree.Members)
+
+	fmt.Println("Get a flat tree")
+	task_list_flat, err := ovc.GetTasks("taskState='Completed'", "", "1", "flat-tree", "", "")
+	if err != nil {
+		fmt.Println("Error getting the task details with status Completed", err)
+	}
+
+	fmt.Println(task_list_flat.Members)
+
+	fmt.Println("Perform Patch operation")
+	task_list_patch, err := ovc.GetTasks("taskState='Running'", "", "", "", "", "")
+
+	for i := 0; i < len(task_list_patch.Members); i++ {
+		if task_list_patch.Members[i].IsCancellable {
+			task_uri := task_list_patch.Members[0].URI
+			task, err = ovc.PatchTask(task_uri.String())
```

Comment:
Add PATCH endpoint in the endpoint support file.

Reply:
Done